custom_components/saj_modbus/hub.py

```diff
@@ -44,6 +45,8 @@ def __init__(
 
         self.inverter_data: dict = {}
         self.data: dict = {}
+        self.limitpower: int = 100
+        self.set_limitpower(100)
```

Comment:
Not sure if this is the right place to set initial power limit as it may block startup if the inverter is offline. This may need to be changed.